protocol/kafka_sarama/v2/coverage.tmp

```diff
@@ -1 +0,0 @@
-mode: atomic
```

Comment:
can we delete this file?

Reply:
Uncertain, but based on the UT script at https://github.com/cloudevents/sdk-go/blob/main/hack/unit-test.sh#L20, it seems that `coverage.tmp` is produced as intermediate output by UT. Deleting it should pose no harm.

Reply:
I can restore it if there are additional uses that I overlooked.